### PR TITLE
feat(OWRT-STQA-207): Add Show All Owners Button

### PR DESCRIPTION
**Pull request by antoheb**
Button to display the list of all the owners added!
- Test implemented
- Test passed
- Comment Added

### CODE REVIEW

File: src/test/java/org/springframework/samples/petclinic/service/AbstractClinicServiceTests.java [hunk 1]
```diff
@@ -62,8 +63,10 @@
 
 @ExtendWith(MockitoExtension.class)
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@ExtendWith(SeleniumExtension.class)
```
Comment (sylveon) [inline src/test/java/org/springframework/samples/petclinic/service/AbstractClinicServiceTests.java]: Should probably remove that if you don't use it

File: src/main/java/org/springframework/samples/petclinic/web/OwnerController.java [hunk 2]
```diff
@@ -97,8 +97,8 @@ public String initFindForm(Map<String, Object> model) {
     public String processFindForm(Owner owner, BindingResult result, Map<String, Object> model) {
 
         // allow parameterless GET request for /owners to return all records
-        if (owner.getLastName() == null) {
-            owner.setLastName(""); // empty string signifies broadest possible search
+        if (owner.getLastName() == "") {
```
Comment (WilliamMeroz) [inline src/main/java/org/springframework/samples/petclinic/web/OwnerController.java:100]: Using owner.getLastName().isEmpty() is better coding practice.

**Review by WilliamMeroz — COMMENTED** [review]
Other than what I have pointed out concerning your empty string, it looks fine. The code is well commented and the logic is easy to follow. Also, as Charles pointed out, you should remove the SeleniumExtension if you are not using it. Well done!

**Review by WilliamMeroz — APPROVED** [review]
All good!

**Review by lucas-cimino — APPROVED** [review]
Nice work, with this code, looking for all owners should be possible!

**Pull request merged**